src/Core/Repositories/ISsoConfigRepository.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Bit.Core.Models.Table;
+
+namespace Bit.Core.Repositories
+{
+    public interface ISsoConfigRepository : IRepository<SsoConfig, long>
+    {
+        Task<SsoConfig> GetBySsoConfigIdAsync(long? id);
```

Comment:
You shouldn't need this since you already get a `...ById()` method from the inherited Repository

Reply:
Interesting, I couldn't get that to work this whole time but now it does.  Will remove.